tutorial/2_single_gpu_ml/alvis-intro-24.10.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/env bash
+
+
+# This script is used for the Open Ondemand portal.
+# You can use it as a reference for creating a custom ~/portal/jupyter/my_jupyter_env.sh file
+
+#SBATCH -A NAISS2024-22-219    # find your project with the "projinfo" command
+#SBATCH -p alvis               # what partition to use (usually not necessary)
+#SBATCH -t 0-00:01:00          # how long time it will take to run
+#SBATCH --gpus-per-node=T4:1   # choosing no. GPUs and their type
+#SBATCH -J modules             # the jobname (not necessary)
+
+module purge
+
+module load PyTorch-bundle/2.1.2-foss-2023a-CUDA-12.1.1
+module load TensorFlow/2.15.1-foss-2023a-CUDA-12.1.1
+module load matplotlib/3.9.2-gfbf-2024a
+
+# You need to have a version of jupyter (e.g. JupyterLab or IPython  from the module system)
+module load JupyterLab/4.2.0-GCCcore-13.2.0
```

Comment:
Note, that these are from different toolchains and are not compatible https://www.c3se.chalmers.se/documentation/modules/#toolchains. You will have to pick the newest common toolchain (in this case I believe it should be 2023a/GCC 12.3.0).

Reply:
Sorry, here no changes, I just noticed that my changes were in other branch. I will fix that and send the pull request again.